src/Storageless/Http/SessionMiddleware.php

```diff
@@ -221,7 +221,7 @@ private function parseToken(Request $request)
      *
      * @return SessionInterface
```

Comment:
We should probably mark this method as `@internal`, unless PHP 7.1 can handle private callbacks somehow...